### ISSUE TITLE
[debian jessie] agent can't install on a vanilla jessie

### ISSUE THREAD

**Issue by LeoCavaille** [DataDog/dd-agent#1411]
```
$ sudo apt-get install datadog-agent
The following NEW packages will be installed:
datadog-agent
0 packages upgraded, 1 newly installed, 0 to remove and 59 not upgraded.
Need to get 0 B/49.6 MB of archives. After unpacking 164 MB will be used.
Selecting previously unselected package datadog-agent.
(Reading database ... 95533 files and directories currently installed.)
Preparing to unpack .../datadog-agent_1%3a5.1.1-546_amd64.deb ...
Unpacking datadog-agent (1:5.1.1-546) ...
Setting up datadog-agent (1:5.1.1-546) ...
insserv: script datadog-agent is not an executable regular file, skipped!
Job for datadog-agent.service failed. See 'systemctl status
datadog-agent.service' and 'journalctl -xn' for details.
invoke-rc.d: initscript datadog-agent, action "restart" failed.
dpkg: error processing package datadog-agent (--configure):
subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
datadog-agent
E: Sub-process /usr/bin/dpkg returned an error code (1)
Failed to perform requested operation on package. Trying to recover:
Setting up datadog-agent (1:5.1.1-546) ...
usermod: no changes
Job for datadog-agent.service failed. See 'systemctl status
datadog-agent.service' and 'journalctl -xn' for details.
invoke-rc.d: initscript datadog-agent, action "restart" failed.
dpkg: error processing package datadog-agent (--configure):
subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
datadog-agent
```

**Comment by SteelPangolin** [comment]
I'm experiencing the same problem except with version 5.2 (specifically `datadog-agent (1:5.2.2-1)`).

**Comment by LeoCavaille** [comment]
@SteelPangolin Thanks for sharing. We'll work on that asap, jessie is soon to be released.
If someone wants to pick it up, it probably has something to do with our packaging and the new _systemd_ default init system in jessie.
see https://github.com/DataDog/dd-agent-omnibus

**Comment by LeoCavaille** [comment]
Referencing #1563

**Comment by LeoCavaille** [comment]
Fixed by the PR above. Released in 5.3.2.

**Comment by SteelPangolin** [comment]
Thanks, @LeoCavaille and @elafarge!
